release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -284,6 +284,27 @@ For more information, see xref:../operators/operator_sdk/osdk-pkgman-to-bundle.a
 
 When running machines in vSphere, you can now set the `minReplicas` value to `0` in the `MachineAutoscaler` resource definition. When this value is set to `0`, the cluster autoscaler scales the machine set to and from zero depending on if the machines are in use. For more information, see the xref:../machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling[MachineAutoscaler resource definition].
 
+[id="ocp-4-8-mco-upgrade-complete"]
+==== MCO waits for all machine config pools to update before reporting the update is complete
+
+When updating, the Machine Config Operator (MCO) now waits for the worker machine config pools (MCP) to finish updating before reporting that the update is complete. Previously, the MCO reported update progress on the status of the master machine config pools, even if the worker pools were not done updating.   
+
+When updating, the Machine Config Operator (MCO) now sets `Upgradeable=False` for any machine config pool that has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported update progress on the status of the master machine config pools, even if the worker pools were not done updating.   
```

Comment:
```suggestion
When updating, the Machine Config Operator (MCO) now reports an `Upgradeable=False` status to the CVO if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported "Upgradeable" status based on only of the master machine config pool, even if the worker pools were not done updating.   
```

Reply:
@wking @yuqi-zhang ptal at my comments

Reply:
"status to the CVO" -> "condition in its ClusterOperator"?  The CVO is one of several potential ClusterOperator consumers.

Reply:
I think the updates look good other than Trevor's comment